src/pages/page-download-success.html

```diff
@@ -148,8 +148,14 @@
              * takes the user to the initiating hashtag page
              */
             _enter() {
-                this.path = sessionStorage.getItem('origin');
-                sessionStorage.clear();
+                if (typeof sessionStorage.getItem('origin') === 'string'
```

Comment:
Why must this be a string?

Reply:
I'm not sure, this is not "new" code I did. It might be from merging one of Xardas' branches into this one. Let's ask him afterwards